docs: Standardize the README (#9)
php-fig/http-factory: README.md

HTTP Factories
==============

This repository holds all interfaces related to [PSR-17 (HTTP Factories)][psr-url].

Note that this is not a HTTP Factory implementation of its own. It is merely interfaces that describe the components of a HTTP Factory.

> You can find [implementations][implementation-url] and [installation instructions][package-url] for the specification on the packagist.

Reviewer: In here the reference to the virtual package is lost in the link

Reviewer 2: ```suggestion
The installable [package][package-url] and [implementations][implementation-url] are listed on Packagist.
```

[psr-url]: https://www.php-fig.org/psr/psr-17/

> [package-url]: https://packagist.org/packages/psr/http-factory

Reviewer 2: ```suggestion
```
